pkgs/desktops/gnome-3/3.20/core/libpeas/default.nix

```diff
@@ -1,13 +1,15 @@
 { stdenv, fetchurl, pkgconfig, intltool, gnome3
-, glib, gtk3, gobjectIntrospection, python, pygobject3
+, glib, gtk3, gobjectIntrospection, python3, python3Packages, ncurses, python2, python2Packages
 }:
 
 stdenv.mkDerivation rec {
   inherit (import ./src.nix fetchurl) name src;
 
+  configureFlags = [ "--enable-python3" "--enable-python2" ];
+
   buildInputs =  [
-   intltool pkgconfig glib gtk3 gobjectIntrospection python pygobject3
-   gnome3.defaultIconTheme
+   intltool pkgconfig glib gtk3 gobjectIntrospection python3 python3Packages.pygobject3 python2
+   gnome3.defaultIconTheme ncurses
```

Comment:
@FRidh the new commited version does exactly that but it still doesn't work:

```
$ /nix/store/xf7p56h7jx0bmqli701a7ghs1r24jx59-totem-3.20.1/bin/totem
(totem:16199): libpeas-WARNING **: Error initializing Python Plugin Loader: PyGObject initialization failed
ImportError: could not import gobject (error was: ImportError("No module named 'gi'",))

(totem:16199): libpeas-WARNING **: Please check the installation of all the Python related packages required by libpeas and try again

(totem:16199): libpeas-WARNING **: Loader 'python3' is not a valid PeasPluginLoader instance

$ nix-store --query --references /nix/store/xf7p56h7jx0bmqli701a7ghs1r24jx59-totem-3.20.1 | grep '\(python\|libpeas\)'
/nix/store/qpwx3rnww9xv24bwqn6pz3cikmmgdngy-libpeas-1.18.0

$ nix-store --query --references /nix/store/qpwx3rnww9xv24bwqn6pz3cikmmgdngy-libpeas-1.18.0 | grep python
/nix/store/w9gwivwr7ih57b4y1dfhv64xy316p5qn-python3-3.5.1
/nix/store/2vxw8wqhkwavr9fj825ihgkk01mv3j13-python3-3.5.1-env
/nix/store/5wyc68zp1qj05ac5xgp18v8qpf259m96-python-2.7.11-env

$ nix-store --query --references /nix/store/2vxw8wqhkwavr9fj825ihgkk01mv3j13-python3-3.5.1-env | grep gobject
/nix/store/l57qb8rir38kfdqjykrnvi1yk1xd070k-pygobject-3.20.0

$ /nix/store/2vxw8wqhkwavr9fj825ihgkk01mv3j13-python3-3.5.1-env/bin/python3
Python 3.5.1 (default, Dec  7 2015, 01:39:13) 
[GCC 5.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import gi
>>> import foo
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named 'foo'
>>> quit()
```

Is it normal that `libpeas` has both python with and without environment as a dependency?